README.md

```diff
@@ -23,29 +24,39 @@ Setup
 
 pa11y-dashboard requires [Node.js][node] 0.10+ and [PhantomJS][phantom]. See the [pa11y][pa11y] documentation for detailed instructions on how to install these dependencies on your operating system.
 
-You'll also need to have [MongoDB][mongo] installed and running. See the [MongoDB install guide][mongo-install] for more information on this. 
+You'll also need to have [MongoDB][mongo] installed and running. See the [MongoDB install guide][mongo-install] for more information on this.
 
-You'll then need to clone this repo locally and install dependencies with `npm install`.
+You'll then need to clone this repo locally and install dependencies with `npm install`. Now we need to add some configuration before we can run the application. We can do this in two ways:
 
-Once you have a local clone, you'll need to copy some sample configuration files in order to run the application. From within the repo, run the following commands:
+### Option 1: Using Environment Variables
+
+Each configuration can be set with an environment variable rather than a config file. For example to run the application on port `8080` you can use the following:
+
+```sh
+PORT=8080 node index.js
+```
+
+The [available configurations are documented here](#configurations).
+
+### Option 2: Using Config Files
+
+You'll need to copy and modify different config files depending on your environment (set with `NODE_ENV`):
 
 ```sh
 cp config/development.sample.json config/development.json
 cp config/production.sample.json config/production.json
 cp config/test.sample.json config/test.json
 ```
 
-Each of these files defines configurations for a different environment. If you're just running the application locally, then you should be OK with just development configurations. The [available configurations are documented here](#configurations).
+Each of these files defines configurations for a different environment. If you're just running the application locally, then you should be OK with just development and test configurations. The [available configurations are documented here](#configurations).
 
-Now that you've got your application configured, make sure you have a server running with the `mongod` command in another terminal window. You can then run in each mode with the following commands:
+Now that you've got your application configured, make sure you have a MongoDB server running with the `mongod` command in another terminal window. you can run in each mode by changing the `NODE_ENV` environment variable:
 
 ```sh
-NODE_ENV=production node index.js   # Run in production
-NODE_ENV=development node index.js  # Run in development
-NODE_ENV=test node index.js         # Run in test
+NODE_ENV=development node index.js
```

Comment:
Was this an intentional deletion?  It seems clearer to me with the variations given as examples rather than just one.